include/dav1d/data.rs

```diff
@@ -67,3 +56,14 @@ impl From<Rav1dData> for Dav1dData {
         }
     }
 }
+
+impl Default for Rav1dData {
```

Comment:
Looks like this just moved but didn't change? Please revert this.

Reply:
Whoops, fixed now.